pkgs/tools/archivers/wimlib/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchurl, makeWrapper
+, pkgconfig, openssl, fuse, libxml2
+, cabextract ? null
+, cdrkit ? null
+, mtools ? null
+, ntfs3g ? null
+, syslinux ? null
+}:
+
+stdenv.mkDerivation rec {
+  version = "1.12.0";
+  name = "wimlib-${version}";
+
+  nativeBuildInputs = [ pkgconfig makeWrapper ];
+  buildInputs = [ openssl fuse libxml2 ntfs3g ];
+
+  src = fetchurl {
+    url = "https://wimlib.net/downloads/${name}.tar.gz";
+    sha256 = "852cf59d682a91974f715f09fa98cab621b740226adcfea7a42360be0f86464f";
+  };
+
+  preBuild = ''
+    substituteInPlace programs/mkwinpeimg.in \
+      --replace '/usr/lib/syslinux' "${syslinux}/share/syslinux"
+  '';
+
+  postInstall = let
+    path = stdenv.lib.makeBinPath  [ cabextract cdrkit mtools ntfs3g syslinux ];
+  in ''
+    for prog in $out/bin/*; do
+      wrapProgram $prog --prefix PATH : ${path}
+    done
+  '';
+
+  doCheck = true;
+
+  preCheck = ''
+    patchShebangs tests
```

Comment:
Correct me if I am wrong, but the extra commands where not required to pass the test suite.

Reply:
some of the tests are supposed to execute `ntfs3g`. The tests are setup in a way that skips execution if the optional binaries are not available.

Reply:
the amount of skipped tests where the same in both cases, so I assume they cannot run in the build sandbox.